recipes/correct-ts-specifiers/src/workflow.ts

```diff
@@ -1,14 +1,14 @@
 import module from 'node:module';
+import type { RegisterHooksOptions } from 'node:module';
 
 import { type Api, api } from '@codemod.com/workflow';
 import type { Helpers } from '@codemod.com/workflow/dist/jsFam.d.ts';
 
 import { mapImports } from './map-imports.ts';
 import type { FSAbsolutePath } from './index.d.ts';
-
 import * as aliasLoader from '@nodejs-loaders/alias/alias.loader.mjs';
 
-module.registerHooks(aliasLoader);
+module.registerHooks(aliasLoader as RegisterHooksOptions);
```

Comment:
very nit: does this work with `satisfies` instead of `as`? if so, that's preferable

```suggestion
module.registerHooks(aliasLoader satisfies RegisterHooksOptions);
```

Reply:
Ah. Go ahead with the `as` and I'll try to look into the type issue tomorrow 🙂

Reply:
Okay !